Formula/fdclone.rb

```diff
@@ -1,8 +1,8 @@
 class Fdclone < Formula
   desc "Console-based file manager"
   homepage "https://hp.vector.co.jp/authors/VA012337/soft/fd/"
-  url "https://hp.vector.co.jp/authors/VA012337/soft/fd/FD-3.01b.tar.gz"
-  sha256 "d66d902cac9d4f64a91d42ceb487a138d544c9fd9cb2961730889cc8830303d4"
+  url "http://www.unixusers.net/src/fdclone/FD-3.01d.tar.gz"
```

Comment:
upstream should be contacted to find out why there's no https available. We generally don't want to do a protocol downgrade if it can be helped

Reply:
There's an email on the site I pulled the link from (https://hp.vector.co.jp/authors/VA012337/soft/fd/), I'll reach out and see what the plans for HTTPS are (and hopefully persuade them to add HTTPS support).  Seems like, based upon the homepage for www.unixusers.net, the website is just starting out, and perhaps doesn't have SSL certificates yet?

Reply:
could be. or he hit some kind of data cap on the other

Reply:
If you hear back, please let us know.